packages/circuit-ui/components/Button/Button.spec.tsx

```diff
@@ -206,5 +206,14 @@ describe('Button', () => {
       const actual = await axe(wrapper);
       expect(actual).toHaveNoViolations();
     });
+
+    it('should have aria-busy and aria-live for a loading button', () => {
+      const wrapper = renderButton(create, {
+        ...baseProps,
+        isLoading: true,
+        loadingLabel: 'Loading...',
+      });
+      expect(wrapper).toMatchSnapshot();
+    });
```

Comment:
It would be easy to miss a change to these attributes in the snapshot in the future. I would be explicit here and check the attributes with testing-library's `.toHaveAttribute(<name>, <value>)` matcher.

Reply:
thanks for that, I have replaced this with the proper attribute checking.